documentation/sphinx/source/api-error-codes.rst

```diff
@@ -176,6 +176,8 @@ FoundationDB may return the following error codes from API functions. If you nee
 +-----------------------------------------------+-----+--------------------------------------------------------------------------------+
 | unknown_tenant                                | 2137| Tenant is not available from this server                                       |
 +-----------------------------------------------+-----+--------------------------------------------------------------------------------+
+| illegal_tenant_access                         | 2138| Tenant information mismatch                                                    |
```

Comment:
This document is intended to capture errors that are allowed to surface up to a user. `illegal_tenant_access` should never actually bubble out, we would only really expect it if someone was using a modified client. I think we can exclude it from this file.